en/docs/administer/configuring-hazelcast.md

```diff
@@ -60,4 +82,19 @@ important to add all the members to the well-known members list in the `
         ```java
         hazelcast.enterprise.license.key=<key>
         ```
-    
\ No newline at end of file
+
+## Why is it mandatory to enable Hazelcast when there are multiple nodes
+
+Applications are scaled horizontally when good performance and reliability are critical. But here comes the problem 
```

Comment:
Can we explain how Hazelcast solves this problem?
@deshankoswatte

Reply:
@nilminiwso2 I have added information on how Hazelcast solves the problem.